docs/export-pdf.md

```
@@ -4,10 +4,19 @@ noIndex: false
-2. In the app editor, choose which blocks should be included in the PDF.
-3. After you are done, open your app in a new tab by clicking on the "View app" button in the right sidebar.
-4. Click on the "Export to PDF" button in the top right corner. This will open your browser's native print dialog to create the PDF. Elements like the top bar or sidebars are hidden in the printed version.
-5. (Optional) Delete the app.
+![Project toolbar with the notebook actions menu](../assets/docs/export-pdf/toolbar-option.webp)
+
+1. Open the notebook you want to export.
+2. Click the three-dot menu in the project toolbar.
+
+![Project context menu with Export as option](../assets/docs/export-pdf/project-context-menu.webp)
+
+3. Hover over **Export as**.
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)